README.md

```diff
@@ -4,7 +4,10 @@
 [![Coverage Status](https://coveralls.io/repos/github/trufflesuite/ganache-core/badge.svg?branch=develop)](https://coveralls.io/github/trufflesuite/ganache-core?branch=develop)
 # Ganache Core
 
-This is the core code that powers the Ganache application and the the Ganache command line tool.
+[Usage](#usage) | [Options](#options) | [Implemented Methods](#implemented-methods) | [Custom Methods](#custom-methods) | [Unsupported Methods](#unsupported-methods) | [Testing](#testing)
```

Comment:
This is so pretty. I wonder if it will render OK in Docker hub? (not that it looks good in Docker right now)

Reply:
Hmm, can we test it on a beta branch (when the time comes)?  Or get a preview somehow

Reply:
Tested it:

![image](https://user-images.githubusercontent.com/187813/54076654-093ecf80-427c-11e9-87fd-ab40de53d6cc.png)

It works! 👍